### PR TITLE
framework: add log info for scheduler

### PR DESCRIPTION
**Pull request by zhangchaoltt**

### CODE REVIEW

File: cyber/scheduler/policy/scheduler_choreography.cc [hunk 1]
```diff
@@ -49,34 +49,28 @@ SchedulerChoreography::SchedulerChoreography() {
   auto cfg_file = GetAbsolutePath(WorkRoot(), conf);
 
   apollo::cyber::proto::CyberConfig cfg;
+  apollo::cyber::proto::ChoreographyConf choreography_conf;
```
Comment (xiaoxq) [inline cyber/scheduler/policy/scheduler_choreography.cc]: Put it inside the `if`.

File: cyber/scheduler/policy/scheduler_choreography.cc [hunk 2]
```diff
@@ -49,34 +49,28 @@ SchedulerChoreography::SchedulerChoreography() {
...
   if (PathExists(cfg_file) && GetProtoFromFile(cfg_file, &cfg)) {
-    proc_num_ =
-        cfg.scheduler_conf().choreography_conf().choreography_processor_num();
-    choreography_affinity_ =
-        cfg.scheduler_conf().choreography_conf().choreography_affinity();
-    choreography_processor_policy_ = cfg.scheduler_conf()
-                                         .choreography_conf()
+    choreography_conf = cfg.scheduler_conf().choreography_conf();
```
Comment (xiaoxq) [inline cyber/scheduler/policy/scheduler_choreography.cc]: Use const reference.

File: cyber/scheduler/policy/scheduler_choreography.cc [hunk 3]
```diff
@@ -49,34 +49,28 @@ SchedulerChoreography::SchedulerChoreography() {
...
+                                         .choreography_processor_prio();
+    ParseCpuset(choreography_conf.choreography_cpuset(), &choreography_cpuset_);
+
+    task_pool_size_ = choreography_conf.pool_processor_num();
+    pool_affinity_ = choreography_conf.pool_affinity();
+    pool_processor_policy_ = choreography_conf.pool_processor_policy();
+    pool_processor_prio_ = choreography_conf.pool_processor_prio();
+    ParseCpuset(choreography_conf.pool_cpuset(), &pool_cpuset_);
+
+    for (auto& thr : choreography_conf.threads()) {
```
Comment (xiaoxq) [inline cyber/scheduler/policy/scheduler_choreography.cc]: const.

File: cyber/scheduler/policy/scheduler_choreography.cc [hunk 4]
```diff
@@ -49,34 +49,28 @@ SchedulerChoreography::SchedulerChoreography() {
...
       inner_thr_confs_[thr.name()] = thr;
     }
 
-    for (auto& task : cfg.scheduler_conf().choreography_conf().tasks()) {
+    for (auto& task : choreography_conf.tasks()) {
```
Comment (xiaoxq) [inline cyber/scheduler/policy/scheduler_choreography.cc]: const.

File: cyber/scheduler/policy/scheduler_choreography.cc (hunk 1 above)
Comment (zhangchaoltt) [reply cyber/scheduler/policy/scheduler_choreography.cc]: done

File: cyber/scheduler/policy/scheduler_choreography.cc (hunk 2 above)
Comment (zhangchaoltt) [reply cyber/scheduler/policy/scheduler_choreography.cc]: done

File: cyber/scheduler/policy/scheduler_choreography.cc (hunk 3 above)
Comment (zhangchaoltt) [reply cyber/scheduler/policy/scheduler_choreography.cc]: done

File: cyber/scheduler/policy/scheduler_choreography.cc (hunk 4 above)
Comment (zhangchaoltt) [reply cyber/scheduler/policy/scheduler_choreography.cc]: done

**Comment by jinghaomiao** [comment]
@zhangchaoltt  Please resolve the conflict.

**Pull request merged**